<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageCureAdjustControl</class>
 <widget class="QWidget" name="ImageCureAdjustControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>365</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Image Curve Adjust</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>9</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel Mode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_channelMode">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>6</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>(All)RGB/Gray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Single Channel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Multi Channel</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>36</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_channel">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>33</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>(Red)R</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(Green)G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(Blue)B</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line_left">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>1</width>
     <height>258</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 160, 164);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_right">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>70</y>
     <width>1</width>
     <height>258</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 160, 164);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_top">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>258</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 160, 164);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_bottom">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>327</y>
     <width>258</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 160, 164);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>289</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Out:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_outNum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>In:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_inNum">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="wd_colorBlack1" native="true">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>329</y>
     <width>256</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="wd_colorBlack2" native="true">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>71</y>
     <width>10</width>
     <height>256</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="wd_colorGreen1" native="true">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>329</y>
     <width>256</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 255, 0, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="wd_colorBlue1" native="true">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>329</y>
     <width>256</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 255, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="wd_colorRed2" native="true">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>71</y>
     <width>10</width>
     <height>256</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="wd_colorGreen2" native="true">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>71</y>
     <width>10</width>
     <height>256</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 255, 0, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="wd_colorBlue2" native="true">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>71</y>
     <width>10</width>
     <height>256</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 255, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="wd_colorRed1" native="true">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>329</y>
     <width>256</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_baseLine">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>3</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BaseLine</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_Histogram">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>29</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogram</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_combine">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>3</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Curve Combine</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_reset">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>33</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
